autonomi/tests/file.rs

```diff
@@ -6,38 +6,77 @@
 // KIND, either express or implied. Please review the Licences for the specific language governing
 // permissions and limitations relating to use of the SAFE Network Software.
 
-#![cfg(all(feature = "files", feature = "fs"))]
+#![cfg(feature = "fs")]
 
 use autonomi::Client;
 use eyre::Result;
+use sha2::{Digest, Sha256};
 use sn_logging::LogBuilder;
+use std::fs::File;
+use std::io::{BufReader, Read};
 use std::time::Duration;
 use test_utils::{evm::get_funded_wallet, peers_from_env};
 use tokio::time::sleep;
+use walkdir::WalkDir;
 
+// With a local evm network, and local network, run:
+// EVM_NETWORK=local cargo test --features="fs,local" --package autonomi --test file
+#[cfg(feature = "fs")]
```

Comment:
This attribute can be removed as we have the top level guard already.

Reply:
Do we?

Reply:
See line 9.